lib/chewy/search/parameters.rb

```diff
@@ -24,30 +38,56 @@ def initialize(initial = {})
         end
       end
 
+      # Compares storages by their values
+      #
+      # @param other [Object] any object
+      # @return [true, false]
       def ==(other)
         super || other.is_a?(self.class) && compare_storages(other)
       end
 
+      # Clones the specified storage, performs the operation
+      # defined by block on the clone.
+      #
+      # @param name [Symbol] parameter name
+      # @yield the block is executed in the cloned storage instance binding
+      # @return [Chewy::Query::Parameters::Value]
       def modify!(name, &block)
         @storages[name] = @storages[name].clone.tap do |s|
           s.instance_exec(&block)
         end
       end
 
+      # Removes specified storages from the storages hash.
```

Comment:
Actually, storage names are given, so it is "removes storages with given names", not "removes given storages" because you are not giving storages, names only., which, in short, is equal to "removes specified storages"